File: src/models/body/PhallicBodyPart.java
package models.body;

import models.entities.Phallic;
import models.items.Item;

/**
 *
 */
public class PhallicBodyPart extends BodyPart implements Phallic {
    private int size;
    private int width;
    public PhallicBodyPart(Item item, String description) {
        super(item, description);
    }

    @Override
    public int getSize() {
        return this.size;
    }

    @Override
    public int getWidth() {
        return this.width;
    }
}

File: Assets/Scripts/body/PhallicBodyPart.cs
using entities;
using items;

namespace body {
	/**
	 *
	 */
	public class PhallicBodyPart : BodyPart, Phallic {
	    private int Size;
	    private int Width;
		public PhallicBodyPart(Item item, string description) : base(item, description) {
	    }

		public int size {
			get { return Size; }
			set { this.Size = value; }
		}

		public int width {
			get { return Width; }
			set { this.Width = value; }
		}
	}
}
